documentation/slate/source/includes/apis.md

```diff
@@ -1716,6 +1716,168 @@ object. Arrivals are ordered chronologically in ascending order.
+and another one of [disruptions](#disruption).
+
+There are inner links between this 2 arrays:
+see the [inner-reference](#inner-references) section to use them.
+
+### Line report object
+``` shell
+#links between objects in a line_reports response
+```
+{
+  "disruptions": [
+    {
+      "status": "active",
+      "id": "17283fae-7dcf-11e8-898e-005056a47b86"
+    },
+    {
+      "status": "active",
+      "id": "140a9970-0c9b-11e8-b2b6-005056a44da2"
+    }
+  ],
+  "line_reports": [
+    {
+      "line": {
+        "links": [],
+        "id": "line:1"
+      },
+      "pt_objects": [
+        {
+          "embedded_type": "stop_point",
+          "stop_point": {
+            "name": "SP 1",
+            "links": [
+              {
+                "internal": true,
+                "type": "disruption",
+                "id": "140a9970-0c9b-11e8-b2b6-005056a44da2",
+                "rel": "disruptions",
+                "templated": false
+              }
+            ],
+          "id": "stop_point:1"
+          }
+        }
+      ]
+    },
+    {
+    "line": {
+        "id": "line:CAE:218",
+        "links": [
+              {
+                "internal": true,
+                "type": "disruption",
+                "id": "17283fae-7dcf-11e8-898e-005056a47b86",
+                "rel": "disruptions",
+                "templated": false
+              }
+        ]
+    },
+    "pt_objects": [
+        {
+            "embedded_type": "line",
+            "line": {
+                "id": "line:CAE:218",
+                "links": [
+                    {
+                        "internal": true,
+                        "type": "disruption",
+                        "id": "17283fae-7dcf-11e8-898e-005056a47b86",
+                        "rel": "disruptions",
+                        "templated": false
+                    }
+                ]
+            }
+        }
+    ]
+}
+]
+}
+```
+Line_reports is an array of some line_report object.
+
+One Line_report object is a complex object, made of a line, and an array
+of [pt_objects](#pt-objects) linked (for example stop_areas, stop_point or network). 
+
+#### What a **complete** response **means**
+
+-   multiple line_reports
+    -   line 1
+          -   stop area concorde > internal link to disruption "green"
+          -   stop area bastille > internal link to disruption "pink"
+    -   line 2 > internal link to disruption "blue"
+          -   network RATP > internal link to disruption "green"
+          -   line 2 > internal link to disruption "blue"
+    -   line 3 > internal link to disruption "yellow"
+          -   stop point bourse > internal link to disruption "yellow"
+-   multiple disruptions (disruption target links)
+    -   disruption "green"
+    -   disruption "pink"
+    -   disruption "blue"
+    -   disruption "yellow"
+    -   Each disruption contains the messages to show.
+
+Details for disruption objects : [disruptions](#disruptions)
```

Comment:
This section is quite messy.

Reply:
So if I follow well, we should have those sections:
* Request
  * Url
  * Parameters
* Response
  * Complete response description
  * Single line report object description

Then we should store everything in the right subsection (and do the same for traffic_report).
This would solve you remark "not talking about params in param section" also.